### DISCUSSION TITLE
Is it possible to have real time reports in Serenity?

### DISCUSSION THREAD

**Discussion by drpetkova** [serenity-bdd/serenity-core#2939] [Q&A] [answered]
Hi, I'm interested if there is a way to generate the serenity report in real time e.g. generate the report after the first test, then update it after every other test (incrementally) instead of generating the report at the end when all the tests have finished.

**Comment by wakaleo** [comment, marked as answer]
I don't think this would be currently possible, as there is a lot of work done to aggregate the test results and to map them to the requirements structure, which can only happen when all the results are known.

**Reply by drpetkova** [reply]
Thank you for the answer. Have you consider having such feature?
